### DISCUSSION TITLE
Wrote a blog post on how to set up Cloudflare Tunnels for a Minecraft Server

### DISCUSSION THREAD

**Discussion by adaliea** [HttpRafa/modflared#26] [Ideas]
A little while ago I wrote a blog post for how to set up Minecraft server using Cloudflare tunnels and this mod: https://dacubeking.com/2024/02/28/Proxying-Minecraft.html. 

Maybe it would be worth it to add this to the ReadMe?

**Comment by HttpRafa** [comment]
I added your blog post to the README under "Other resources"
